### PR TITLE
[ConvertLayout] Resize operator

### PR DESCRIPTION
**Pull request by anijain2305**
As Title
@comaniac

### CODE REVIEW

File: src/relay/op/image/resize.cc [hunk 1]
```diff
@@ -33,6 +33,23 @@ namespace relay {
...
 TVM_REGISTER_NODE_TYPE(ResizeAttrs);
 
+Array<Array<Layout> > ResizeInferCorrectLayout(const Attrs& attrs,
+                                               const Array<Layout>& new_in_layouts,
+                                               const Array<Layout>& old_in_layouts,
+                                               const Array<tvm::relay::Type>& old_in_types) {
+  // NOTE: Discard "const" qualifier here.
+  ResizeAttrs* params = const_cast<ResizeAttrs*>(attrs.as<ResizeAttrs>());
+
+  if (new_in_layouts.defined()) {
```
Comment (comaniac) [inline src/relay/op/image/resize.cc:43]: When I was working on the `transpose`, I found that it's possible to have `new_in_layouts` defined but `new_in_layouts[0]` undefined. I'm not sure if it's a case in `resize` tho.

**Review by comaniac — APPROVED** [review]
Overall LGTM. Just a small check.

**Comment by comaniac** [comment]
@anijain2305 please fix the CI error that seems failed at an OD model.

**Comment by comaniac** [comment]
@anijain2305 gentle ping for this PR.

**Comment by masahi** [comment]
https://github.com/apache/tvm/pull/8205

**Pull request closed without merging**